synphot/units.py

```diff
@@ -295,6 +295,18 @@ def validate_unit(input_unit):
             output_unit = THROUGHPUT
         elif input_unit_lowcase == 'jy':
             output_unit = u.Jy
+        elif input_unit_lowcase == "flam":
```

Comment:
If there are anymore weird units that STScI uses in any table column `synphot` needs to parse, we have to hardcode the exception rule here, or fix the data.

Now that we go though `astropy.table.QTable`, it parses all the columns. AFAIK, there is no way to skip. This is the price we pay for a more flexible table format.